### ISSUE TITLE
Infoscreens

### ISSUE THREAD

**Issue by Davnik** [HeavensGate/Eternal#262]
Wall-mounted machines which, when powered?, provide a message, either as a custom description or a paper pop-up that you can write on without a pen.  Easy interface lore delivery system.

Mandatory goals:
~It needs to be sprited. It should be a dimly lit grey rectangle which fits for walls, with a holoscreen on it, perhaps with some minor details of a touch-screen interface, perhaps a simple animation, perhaps an icon state difference between written and blank. It at least needs to be a grey border with a glass screen, preferably displaying the glass is over more greyness, so a kind of dim grey screen if not transparent.
~It needs to present at least a custom description. I would prefer if paper code could be ported in, or at least pop-up textbox. In the event of paper code, allow it to be written on without a pen, because on-screen keyboard.
~Power is optional, as it can presumably run on batteries near-indefinitely, (and we want lore delivered even if the station is kill.)

Stretch goals: 
http://pastebin.com/Q3bZHrKw

**Comment by Davnik** [comment]
Once these are implemented, a new issue will arise in gate/writing to have things ready to put on infoscreens, and an issue will then arise in mapping to actually place the buggers where lore needs delivering.

**Comment by Mewchild** [comment]
@Finriv @Puckaboo2

**Comment by Finriv** [comment]
So basically the AI screens which the message can be set. and scroll. Just instead anyone can interact with it like paper

**Comment by Finriv** [comment]
So basically the AI screens which the message can be set. and scroll. Just instead anyone can interact with it like paper

**Comment by Finriv** [comment]
So basically the AI screens which the message can be set. and scroll. Just instead anyone can interact with it like paper

**Comment by Finriv** [comment]
So basically the AI screens which the message can be set. and scroll. Just instead anyone can interact with it like paper

**Comment by Davnik** [comment]
Yeah basically. But, for messages longer than four words. And mostly for mapper use. I don't even know if crew should be able to edit them, but... it could have roleplay uses. Maybe allow players to add an additional, separate message, like a newscaster feed...
